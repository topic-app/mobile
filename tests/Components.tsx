import React from 'react';
import renderer from 'react-test-renderer';

import { Avatar } from '@components';

test('avatar renders correctly', () => {
  const tree = renderer.create(<Avatar name="Jason McDuck" />).toJSON();
  expect(tree).toMatchSnapshot();
});
