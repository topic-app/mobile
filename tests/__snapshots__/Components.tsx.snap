// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`avatar renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#6200ee",
        "borderRadius": 32,
        "height": 64,
        "width": 64,
      },
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
      Object {},
    ]
  }
>
  <Text
    numberOfLines={1}
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
          "textAlign": "left",
        },
        Array [
          Object {
            "textAlign": "center",
            "textAlignVertical": "center",
          },
          Object {
            "color": "#ffffff",
            "fontSize": 32,
            "lineHeight": 64,
          },
          undefined,
        ],
      ]
    }
  >
    JM
  </Text>
</View>
`;
